### ISSUE TITLE
Inititial node positions are not being restored.

### ISSUE THREAD

**Issue by mikekucera** [cytoscape/enrichment-map-webapp#210]
**Describe the bug**

When clicking the "Restore Network to Initial Layout" button, it actually restores the network to the first edit that was made after the initial layout.

**To Reproduce**

1) Create a network
2) Select one node and move it someplace where it was obvious it was moved
3) Wait 10 seconds (saving positions is debounced)
4) Move another node
5) Wait 10 seconds
6) Click the "restore Network to Initial Layout button"
- The network is restored to the state it was in after step 2, it should be restored to the state in step 1.

_Add any other context about the problem here._

The positions are saved when a 'position' event is fired. The initial layout does not fire this event so the node positions are not saved until the first edit is made.

- Note: Why are the initial node positions being saved at all? (I can't remember). The layout in controller.applyLayout() is deterministic, so running it again should produce the same results.

**Comment by maxkfranz** [comment]
> Why are the initial node positions being saved at all? (I can't remember). The layout in controller.applyLayout() is deterministic, so running it again should produce the same results.

Probably re. the PRNG seed. We were considering other layouts and removing the Math.random monkey patching. There's also that aspect ratio layout bug we encountered. 

In any case, one advantage of saving the initial positions is performance. FD layouts can be expensive on large networks. 

If saving isn't needed right now, it may be best to just disable it for now -- rather than deleting the feature. We may have to revisit this if and when we do more performance tuning.

**Comment by mikekucera** [comment]
> There's also that aspect ratio layout bug we encountered.

I think that was it. I was getting a different initial layout if the window size changed.

**Comment by maxkfranz** [comment]
That should be fixed in the latest lib version. We can just upgrade if we haven't already.

**Comment by mikekucera** [comment]
That would still require some work to determine the best initial bounding box, and remove the code for the workaround. I'll open a separate issue.

**Comment by maxkfranz** [comment]
Sounds good. 

Of note: It might suffice to specify a giant bounding box.
